package.json

```diff
@@ -75,10 +77,12 @@
     "redis-clustr": "^1.7.0",
     "ulid": "^2.3.0",
     "winston": "^3.2.1",
-    "yaml": "^1.10.0"
+    "yaml": "^1.10.0",
+    "snyk": "^1.395.0"
```

Comment:
Are we sure we want this dependency to be shipped in our application?

Reply:
cc @gunzip @francescopersico

Reply:
why not ? anyway we cannot ship it until we upgrade to nodejs12

Reply:
I could understand if it was a `devDependencies`, but why do we need it in production? Isn't it a development tool?

Reply:
yes it should go into devdependencies, I don't know if Snyk is smart enought to realize it if you do that manually

Reply:
https://support.snyk.io/hc/en-us/articles/360000897937-Why-is-Snyk-added-into-my-production-dependencies-

Reply:
I stil don't get it. Why it's not required here https://github.com/pagopa/io-functions-admin/pull/75?

Reply:
don't know the internals but this is a 'Fix to vulnerability' while the other PR is un upgrade